src/shared/server-config.toit

```diff
@@ -59,13 +58,16 @@ abstract class ServerConfig:
   */
   abstract to-service-json [--der-serializer] -> Map
 
+  /**
+  Computes a unique key that can be used for caching.
+  */
+  abstract compute-cache-key_ -> string
+
   /**
   A unique key that can be used for caching.
   */
   cache-key -> string:
-    if not cache-key_:
-      hash := sha1.sha1 (ubjson.encode (to-json --der-serializer=: it))
-      cache-key_ = "$(base64.encode --url-mode hash)-$name"
```

Comment:
You dropped the url-mode. Not sure that was intentional.

Reply:
Not intentional and needed...
done.